pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight-cgap"
-version = "2.1.4"
+version = "2.2.0.1b58"
```

Comment:
should you not be starting a new beta series, like `2.1.4.1b01`?

Notwithstanding the fact that the `CHANGELOG.rst` is OK with having a guess at 2.2.0, I do not recommend basing the beta number on your target version but on the version you are basing from, so on `2.1.4` not `2.2.0` since someone might beat you to the new number and then it's confusing, but also because the only thing that matters is to be above the base version (hence `2.1.4.1` + a beta designator.). The changelog consistency verifier will ignore consistency errors when the actual version is a beta version.